src/components/AddExecomDialog.js

```diff
@@ -96,23 +94,12 @@ Date.prototype.toDatetimeLocal = function toDatetimeLocal() {
       }
     }, [props.data]);
   
-
-    const handleFileInput = async file => {
-      const data = new FormData();
-      data.append('image', file);
-      try {
-        const res = await axios.post(`https://api.imgbb.com/1/upload?key=${process.env.REACT_APP_IMGBB_API_KEY}`, data);
-        setValues({
-          ...values,
-          imagepath: res.data.data.display_url,
-        });
-        setLoading(v => !v);
-      } catch (err) {
-        console.log(err);
-        setLoading(v => !v);
-      }
+    const {isLoading, error, uploadImage } = useImageUploader();
+    const handleFileInputChange = async file => {
+      uploadImage(file);
+      if(error) console.log(error.message);
```

Comment:
can we display small neat error message in case if the upload fails?

```javascript
    {error && <p>Error: {error.message}</p>}
```
similar to this pattern, feel free to customise it